less/cards.less

```diff
@@ -625,15 +644,14 @@
           width:  150px;
         }
         .area  {
-          width:  10%;
-          max-width: 120px;
+          width:  20%;
```

Comment:
two spaces between value

Reply:
Concerning the area modifications, I wasn't able to understand where your modifications apply either, sorry (still this mess in the CSS :sob:). Could you provide more details or some screenshots of before/after?

Reply:
It's for list to equilibrate the content. i provide some screenshot